aspnetcore/blazor/components.md

```diff
@@ -511,6 +511,129 @@ Use `EventCallback` and `EventCallback<T>` for event handling and binding compon
 
 Prefer the strongly typed `EventCallback<T>` over `EventCallback`. `EventCallback<T>` provides better error feedback to users of the component. Similar to other UI event handlers, specifying the event parameter is optional. Use `EventCallback` when there's no value passed to the callback.
 
+## Chained bind
+
+A common scenario when creating nested components is the desire to bind a property:
+
+* To a page element in a component.
+* Across nested components.
+
+This scenario is called a *chained bind* because multiple levels of binding occur simultaneously.
+
+The following `PasswordField` component (*PasswordField.razor*):
+
+* Binds an `<input>` element to a `Password` property.
```

Comment:
I think this merits some explanatory text to point out that you don't use the standard `@bind` syntax in this case. You need to instead go level lower to specify the event handler and value yourself.

Reply:
Can it just say 'sets the value' then? That would remove the word "bind" from it.

Reply:
> * Sets the value of an `<input>` element to a `Password` property.

Reply:
It both sets the value and listens to the desired event.

Reply:
Is the "listens" part the 2nd bullet tho?

Reply:
The important thing is to say that you can't use `@bind` when implementing a chained bind.

Reply:
Ok.

Since this two-bullet sequence is only trying to describe what the code in the following example does, I'll try adding it a bit earlier ... something like ...

> Chained bind can't be implemented with `@bind` syntax. Chained bind is created using the property with an event handler.

Reply:
Yeah, "Chained bind can't be implemented with `@bind` syntax *on the page element*...". The consume of the component can of course use `@bind` with the parameter.